components/avatar/avatar.css

```diff
@@ -0,0 +1,96 @@
+.avatar-container{
+    height:auto;
+    width:auto;
+    display:flex;
+    flex-direction:column;
+    align-items:center;
+    justify-content:center;
+  }
+  .avatar-sm, .avatar-md, .avatar-lg{
+    border:1px solid var(--dark-color);
+    width:12.5rem;
+    height:12.5rem;
+    border-radius:50%;
+    object-fit:cover;
+    
+  }
+  .Avatars{
```

Comment:
try to follow one naming convention, either Avatar or avatar.

Reply:
sure